pages/ref/core-cells.mdx

```diff
@@ -462,6 +462,27 @@ let s: Slice = beginCell().storeBool(true).asSlice();
 let fizz: Bool = s.loadBool(); // true
 ```
 
+## Slice.loadBool
```

Comment:
but we just merged https://github.com/tact-lang/tact-docs/pull/270

Reply:
Hmm, seems like the older one was brought over. Tried to fix with rebasing from the main, still there :/